indico/modules/events/contributions/util.py

```diff
@@ -148,7 +148,13 @@ def filter_report_entries(self, query, filters):
         if not filters.get('items'):
             return query
         criteria = []
-        # TODO: Handle contribution status
+        if 'status' in filters['items']:
+            filtered_statuses = filters['items']['status']
+            if 'scheduled' in filtered_statuses:
+                criteria.append(db.or_(Contribution.is_scheduled))
+            if 'unscheduled' in filtered_statuses:
+                criteria.append(db.or_(~Contribution.is_scheduled))
```

Comment:
it looks like you meant to append it to criteria and then use `db.or_(*criteria)` afterwards for the query